### ISSUE TITLE
Gemfile

### ISSUE THREAD

**Issue by mattb2401** [projectfedena/fedena#32]
No Gemfile was found. please git the gem file

**Comment by tachyons** [comment]
This is rails 2.3 project, So gemfile isn't necessary

**Comment by mattb2401** [comment]
i have a rails 4 installed so when i run bundle install i get the error No Gemfile Found

**Comment by tachyons** [comment]
@mattb2401 use [this](http://projectfedena.org/install) guide for installing

**Comment by praveen4zenith** [comment]
I am following same installation guide and on below command I am getting error No GemFile Found:

bundle install --local

**Comment by sanjeevkse** [comment]
Yes. We are getting the same error too. "Could not locate Gemfile"
